docs/decisions/0002-proctoring-email-template-overrides.rst

```diff
@@ -0,0 +1,48 @@
+Proctoring Email Template Overrides
+-----------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+edX sends a learner email(s) with the status of their proctored exam attempt after the learner
+has submitted their proctored exam. The content(s) of these emails is specified by `email templates`_
+in the edx-proctoring repository. These emails may include contact information in the event that a learner
+has further questions about their proctored exam status.
+
+.. _`email templates`: https://github.com/edx/edx-proctoring/tree/master/edx_proctoring/templates/emails
+
+Proctoring services are provided by edX by integrating with third-party proctoring solutions.
+
+Depending on the proctoring provider used for a proctored exam, the contents of an email sent to a learner may need to refer to provider specific information, such as contact information, or have a different structure.
```

Comment:
`the contents of an email sent to a learner may need to contain provider specific information,`